site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -100,6 +100,7 @@ The following tables lists the configurable parameters of the Agones chart and t
 |----------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|
 | `agones.crds.install`                                    | Install the CRDs with this chart. Useful to disable if you want to subchart (since crd-install hook is broken), so you can copy the CRDs into your own chart.                                                                                                                                                    | `true`                                    |
 | `agones.crds.cleanupOnDelete`                            | Run the pre-delete hook to delete all GameServers and their backing Pods when deleting the helm chart, so that all CRDs can be removed on chart deletion                                                                                                                                                         | `true`                                    |
+| `agones.crds.cleanupJobTTL`                              | The number of seconds for Kubernetes to delete the associated Job and Pods of the pre-delete hook after it completes, regardless if the Job is successful or not. Set to `0` to disable cleaning up the Job or the associated Pods.                                                                              | `60`                                      |
```

Comment:
This looks great - quick thing, can you make a copy of the table add `feature` codes around the new and old version?

See: https://agones.dev/site/docs/contribute/#within-a-page

We publish the docs on merge with `main`, so we have a way to keep old features hidden until they are ready in the next release.

Lemme know if you have any questions.

Reply:
I think I implemented this properly; let me know if I missed something!

Reply:
Looks good! Will see the preview when the build finishes! 👍🏻